packages/pratt-parser/pratt-parser.0.1.0-0/url

```diff
@@ -0,0 +1 @@
+archive: "https://github.com/fold-lang/pratt-parser/archive/master.zip"
```

Comment:
Is it really what you want?  The code that will be downloaded for version 0.1.0 will be different depending of the state of your repository — definitely not good.  Add a tag first to your repo and use it to make a release.